attestation-service/docs/policy.md

```diff
@@ -30,13 +19,25 @@ These dimensions can be used to return a detailed, but generic description of th
 More information about these trust claims, including what the numerical values of the claims represent,
 can be found [here](https://datatracker.ietf.org/doc/draft-ietf-rats-ar4si/).
 
-AS policies will be provide `data` and `input`. `data.reference` points to a dictionary of reference
-values provided by the RVPS.
+AS policies will be provide `input`.
 
 `input` are the TCB claims generated by the verifier.
 
 See the [default policy](../src/ear_token/ear_default_policy_cpu.rego) for an example.
 
+> [!NOTE]
+> We now provide a new extension in AS policy. If any field or value should be fetched from RVPS,
+> we can use function `query_reference_value(key_name)` in the rego to get reference value with key
+> id `key_name` from RVPS. If no key is found in RVPS, the policy enforcement will fail.
```

Comment:
To be precise, the clause where the extension is used will evaluate to false/null.